### DISCUSSION TITLE
Timezone dagFactory with yaml

### DISCUSSION THREAD

**Discussion by EmilaineBorato** [apache/airflow#33258] [General]
Hey,

Have a dynamic application to create dags in YAMLS, these dags using DagFactory, execution with local time timezone, even if I change the environment variables applying to timezone=UTC.

For example, the cron is 10AM and the run is cron -3 hours, 07AM.
How to fix it?
Has anyone gone through something similar?

I'm using a clustered environment. Bitnami airflow image, version 2.5.2.

![image](https://github.com/apache/airflow/assets/28653612/59c968a7-b6f3-48c5-a543-f5e0743f46e7)

**Comment by hussein-awala** [comment]
How do you define the start_date of your dags? Do you provide a timezone with the start_date date? because this timezone overrides the default timezone defined in Airflow configurations.

**Comment by EmilaineBorato** [comment]
Yes, I set it when creating a dag, but even then it follows the local timezone  -3.

![image](https://github.com/apache/airflow/assets/28653612/1de33d78-0f83-416e-8b9b-e27f067ab176)

![image](https://github.com/apache/airflow/assets/28653612/c1accfb6-c036-4afc-9298-5afd192b0d44)
